src/replication.c

```diff
@@ -1510,57 +1510,118 @@ get_replica_stats_json(replica_t *replica, struct json_object **jobj)
 	*jobj = j_stats;
 }
 
+static int
+get_cv_status(spec_t *spec, int replica_cnt, int healthy_replica_cnt)
+{
+	if (replica_cnt < spec->consistency_factor)
+		return 1;
+	if (healthy_replica_cnt < spec->consistency_factor)
+		return 2;
+	if (healthy_replica_cnt == spec->replication_factor)
+		return 4;
+	return 3;
+}
+
+static const char *
+status_to_str(int status)
```

Comment:
@vishnuitta we can avoid this function and use `char *` `return value` in `get_cv_status`.